### PR TITLE
Create INS1019DabraHalleloEndaSamuel.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Dabra Hallelo Enda Abuna Samuel

### CODE REVIEW

File: new/INS1019DabraHalleloEndaSamuel.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <text>
+        <body>
+            <listPlace>
+                <place type="monastery" subtype="institution">                                
+                        <placeName xml:id="n1" xml:lang="gez">ደብረ፡ ሃሌሎ፡ እንዳ፡ አቡነ፡ ሳሙኤል፡</placeName>
+                    <placeName corresp="#n1" xml:lang="en">Dabra Hāllelo ʾƎnda ʾAbuna Sāmuʾel</placeName>
+                    <placeName xml:id="n2" xml:lang="gez">ደብረ፡ ሃሌሎ፡</placeName>
+                    <placeName corresp="#n2" xml:lang="en">ደብረ፡ ሃሌሎ፡</placeName>
+                    <placeName xml:id="n3" xml:lang="gez">ደብረ፡ ሃሌሉያ፡</placeName>
+                    <placeName corresp="#n3" xml:lang="en">Dabra Hāllelo</placeName>
```
Comment (CarstenHoffmannMarburg) [inline new/INS1019DabraHalleloEndaSamuel.xml:54]: Dabra Hālleluyā

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: new/INS1019DabraHalleloEndaSamuel.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        <revisionDesc>
+            <change who="DN" when="2024-09-10">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="monastery" subtype="institution">                                
+                        <placeName xml:id="n1" xml:lang="gez">ደብረ፡ ሃሌሎ፡ እንዳ፡ አቡነ፡ ሳሙኤል፡</placeName>
+                    <placeName corresp="#n1" xml:lang="en">Dabra Hāllelo ʾƎnda ʾAbuna Sāmuʾel</placeName>
```
Comment (karljonaskarlsson) [inline new/INS1019DabraHalleloEndaSamuel.xml:50]: should it be ʾƎndā?

File: new/INS1019DabraHalleloEndaSamuel.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                    <placeName xml:id="n2" xml:lang="gez">ደብረ፡ ሃሌሎ፡</placeName>
+                    <placeName corresp="#n2" xml:lang="en">ደብረ፡ ሃሌሎ፡</placeName>
```
Comment (karljonaskarlsson) [inline new/INS1019DabraHalleloEndaSamuel.xml:52]: should the English be transcribed "Dabra Hāllelo"?

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]

File: new/INS1019DabraHalleloEndaSamuel.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/INS1019DabraHalleloEndaSamuel.xml:52]: yes. Thanks!

**Pull request merged**